ocs_ci/ocs/container.py

```diff
@@ -0,0 +1,311 @@
+
+import logging
+import concurrent.futures
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.exceptions import CommandFailed
+
+from tests.helpers import (create_unique_resource_name,
+                           get_admin_key, get_cephfs_name,
+                           get_cephfs_data_pool_name)
+
+log = logging.getLogger(__name__)
+
+
+class ResourceStor(object):
+    """
+    Container Storage class
```

Comment:
A more in-depth description of this class would be nice, given it's scope. Instructions on how to properly instantiate, use, and clean up(since this doesn't happen automatically) should be included.